### PR TITLE
ACM-4185 Show searchbar suggestions when operator is used in search query

### PR DESCRIPTION
**Pull request by zlayne**

### CODE REVIEW

File: frontend/src/routes/Home/Search/SearchPage.tsx [hunk 1]
```diff
@@ -152,31 +151,40 @@ function RenderSearchBar(props: {
...
-          _.get(searchSchemaResults, 'data.searchSchema.allProperties', []),
+          _.get(searchSchemaData, 'searchSchema.allProperties', [
+            'name',
+            'namespace',
+            'label',
+            'kind',
+            'cluster',
+            'apigroup',
+            'created',
+            'Loading more...',
```
Comment (KevinFCormier) [inline frontend/src/routes/Home/Search/SearchPage.tsx:173]: Looks like "Loading more..." was previously translated and now is not?

Comment (KevinFCormier) [reply frontend/src/routes/Home/Search/SearchPage.tsx:173]: Or rather "Loading..." was previously.

Comment (zlayne) [reply frontend/src/routes/Home/Search/SearchPage.tsx]: Added the `Loading more...` string to translations.

**Pull request merged**